client/src/components/Reviews.jsx

```diff
@@ -1,49 +1,49 @@
 import React from 'react';
 import User from './User.jsx';
 import Stars from './Stars.jsx';
+import style from './style.css';
 
 const Reviews = (props) => (
-  <div className="review">
+  <div className={style.review}>
     { props.reviews.map((review) => {
       return (
-        <div className="review-container">
-          <div className="user-container">
+        <div className={style.reviewContainer}>
+          <div className={style.userContainer}>
             <User username={review.username ? review.username : "OpenTable Diner"} 
               city={review.city ? review.city : ""}
               state={review.state ? review.state : ""}/>
           </div>
-          <div className="review-text-container">
-            <div className="star-container">
-              <Stars starRating={review.overall_rating} /> 
+          <div className={style.reviewContentContainer} style={{flex: 4, marginTop: "30px", paddingBottom: "20px"}}>
```

Comment:
Why is there an inline style here? If it can be moved into the reviewContentContainer class, you should do so